docs/physics/spectrum/basic.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>What is the advantage of using <code>quantity_linspace</code> instead of <code>np.linspace</code>? Both generate arrays with astropy quantities.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>